### PR TITLE
Feature/sem 023

### PR DESCRIPTION
**Pull request by ELVISrodrigo123**
feature/SEM-023
https://ditmar.github.io/backoffice-openscience/branch-SEM-023/?path=/story/ui-components-atoms-tablabel--small

### CODE REVIEW

**Comment by DanX3N0** [comment]
check the font

File: src/ui/components/atoms/TabLabel/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@import '../../../../globals/ads-variables.scss';
+@import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
```
Comment (Ditmar) [inline src/ui/components/atoms/TabLabel/styles.module.scss:2]: please use another approach to embed the font. Use this approach 
https://github.com/Ditmar/OpenScience/blob/main/src/globals/_fonts.scss

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice work only a few comments there

File: src/globals/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    font-weight: normal;
+    font-weight: bold;
+    src: url('/fonts/Lato-Bold.woff2') format('woff2');
+
+    @font-face {
+        font-display: swap;
+        font-family: Lato;
+        font-style: normal;
+        font-weight: normal;
+        src: url('/fonts/Lato-Regular.woff2') format('woff2');
```
Comment (Ditmar) [inline src/globals/_fonts.scss:14]: solo copiaste el archivo del otro proyecto, esto esta mal tienes que adpatarlo al tipo de fuente que deseas importar

**Review by Ditmar — CHANGES_REQUESTED** [review]
lee el comentario

**Review by Ditmar — CHANGES_REQUESTED** [review]
Please check the rebase something is bad 
image
you are pushing changes that are not yours

**Review by Ditmar — APPROVED** [review]

**Pull request merged**